app/models/manageiq/providers/autosde/inventory/persister/storage_manager.rb

```diff
@@ -65,6 +65,12 @@ def initialize_inventory_collections
         :ems_id => ->(persister) { persister.manager.id }
       )
     end
+    # wwpn candidates
+    add_collection(storage, :wwpn_candidates) do |builder|
+      builder.add_default_values(
+        :ems_id => ->(persister) { persister.manager.id }
+      )
```

Comment:
This should be handled by the `add_common_default_values` in the core persister definitions https://github.com/ManageIQ/manageiq/pull/21363/files#diff-f623f973e49e35b8e6526b8a41057bb3707c90ed14623c932e38054ec081fb3dR34-R36

It isn't harming anything so I'll merge with this here but can you put in a follow-up PR to remove this?